extensions/scheduler/common/src/main/java/io/quarkus/scheduler/common/runtime/DefaultInvoker.java

```diff
@@ -14,11 +15,18 @@ public CompletionStage<Void> invoke(ScheduledExecution execution) throws Excepti
         if (requestContext.isActive()) {
             return invokeBean(execution);
         } else {
+            // 1. Activate the context
+            // 2. Store the state (which is basically a shared Map instance)
+            // 3. Destroy the context correctly when the returned stage completes
```

Comment:
I'm not a big fan of this comments. You are also not mentioning the most important thing: you don't expect an exception from `invokeBean` so we don't need a try-catch

Reply:
> I'm not a big fan of these comments. 

Why? I find them often very useful. Such a summary is usually easier to read then the code that follows. Although it gets very often outdated (which is another story).

>  You are also not mentioning the most important thing: you don't expect an exception from invokeBean so we don't need a try-catch

Well, I wouldn't say it's the most important thing but it's an important detail that should be included. We could also add a catch block to be on the safe side.

Reply:
Just added the catch block.

Reply:
Yeah, it was too nitpicking. Forget about it. Thanks